docs/3.Custom Policies/YAML Custom Policies.md

```diff
@@ -308,3 +308,46 @@ definition:
 
 [See all examples of Custom Policies in code](https://www.checkov.io/3.Custom%20Policies/Examples.html)
 
+## Supported Frameworks
+
+### Bicep
+All resources can be referenced under `resource_types`.
+Any kind of connection between resources is supported
```

Comment:
@gruebel That's a separate PR I belive :)

Reply:
I just wanted to add some kind of docs, so it is more clear what to expect. This would need further details for sure 😄